### DISCUSSION TITLE
Add suggestion to join Senior+, Mid-level, Early-career private slack channels to VirtualCoffee.io slackbot welcome message

### DISCUSSION THREAD

**Discussion by ebanner** [Virtual-Coffee/virtualcoffee.io#1154] [New Ideas]
It would be cool to have it be suggested to join either the `Senior+`, `Mid-level`, or `Early-career` private slack channels when you first join the slack

![image](https://github.com/Virtual-Coffee/virtualcoffee.io/assets/2068912/d2f98034-8082-44ee-9b7d-5d9b0c50ed6a)

There was a [flurry of interest](https://virtual-coffee-group.slack.com/archives/C014ABH7XU5/p1712756765825779) when it recently got mentioned that these channels exist, so I think it would be good to let folks know about these channels when they first join VC

I made a [PR](https://github.com/Virtual-Coffee/webhooks/pull/18) that proposes to add the following message

> 📝 We have three private channels — Senior+, Mid-level, Early-career — for peer-to-peer conversation to support and nurture conversations specific to your career level. If you’d like join one, then reach out to Meg. If you need help choosing between channels, send a DM to Meg and you can discuss it together.

**Comment by meg-gutshall** [comment]
Thanks for bringing this up, @ebanner! You're correct in that there was a flurry of interest on that post. This could be due to new members not knowing about the channels, existing members missing the previous Slack posts about them, or the lack of organic discovery since they're private channels.

You make an excellent suggestion, however, I worry about packing too much information into the welcome message. The abundance of channels on our Slack can very easily overwhelm people! 😅

We have a [Virtual Coffee Slack Channels Guide](https://virtualcoffee.io/resources/virtual-coffee-handbook/guides-to-virtual-coffee/slack-channels-guide) on our website, though I believe it's not currently up to date. I think a good alternative would be to send a follow-up Slack message a week or two after the welcome message and point new members toward this page on our website as well as any other helpful resources within Slack, the website, our GitHub Org, or even our YouTube channel.

I'm not fully apprised of the present new member onboarding process yet, so I'd be interested to hear thoughts from the other @Virtual-Coffee/maintainers on this.
